web/modules/custom/talkcodemodule/talkcodemodule.css

```diff
@@ -0,0 +1,53 @@
+section#block-talkcodemoduleblock {
+    margin: 0 auto;
+    width: 100%;
+    height: 600px;
+    background-color: #000;
+    background: url(bird_out.png) #000 no-repeat;
```

Comment:
@brandtziegler  It looks like only 1 image is used (bird_out.png), but many images were committed to this pull request. I'm going to merge the PR so that we can continue to next tasks, but the additional images should be removed from the codebase in a subsequent commit. This goes back to always aiming for granular/surgical commits.